README.md

```diff
@@ -13,6 +15,8 @@
  ]
 ```
 
-# Method
+2. `composer require danhariton/data-boxes`
```

Comment:
Use lower-case names only as composer is sensitive to it (maybe only on Linux, not sure).